flink-staging/flink-gelly/src/main/java/org/apache/flink/graph/Graph.java

```diff
@@ -1118,30 +1119,52 @@ public boolean filter(Edge<K, EV> edge) {
 	}
 
 	/**
-	 * Create a Vertex-Centric iteration on the graph.
-	 * 
+	 * Runs a Vertex-Centric iteration on the graph.
+	 * No configuration options are provided.
+
 	 * @param vertexUpdateFunction the vertex update function
 	 * @param messagingFunction the messaging function
 	 * @param maximumNumberOfIterations maximum number of iterations to perform
-	 * @return
+	 * 
+	 * @return the updated Graph after the vertex-centric iteration has converged or
+	 * after maximumNumberOfIterations.
 	 */
-	public <M> VertexCentricIteration<K, VV, M, EV> createVertexCentricIteration(
+	public <M> Graph<K, VV, EV> runVertexCentricIteration(
 			VertexUpdateFunction<K, VV, M> vertexUpdateFunction,
 			MessagingFunction<K, VV, M, EV> messagingFunction,
 			int maximumNumberOfIterations) {
-		return VertexCentricIteration.withEdges(edges, vertexUpdateFunction,
-				messagingFunction, maximumNumberOfIterations);
+
```

Comment:
Function call could be forwarded to the configured `runVertexCentricIteration` to reduce code duplication.